packages/meditrak-server/src/kobo/startSyncWithKoBo.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import keyBy from 'lodash.keyby';
+import winston from 'winston';
+import { DataBroker } from '@tupaia/data-broker';
+import { generateId } from '@tupaia/database';
+import { reformatDateStringWithoutTz } from '@tupaia/utils';
+
+const PERIOD_BETWEEN_SYNCS = 10 * 60 * 1000; // 10 minutes between syncs
+const SERVICE_NAME = 'KoBo';
+
+export async function startSyncWithKoBo(models) {
+  if (process.env.KOBO_SYNC_DISABLE) {
+    // eslint-disable-next-line no-console
+    console.log('KoBo sync is disabled');
+  } else {
+    const dataBroker = new DataBroker();
+    setInterval(() => syncWithKoBo(models, dataBroker), PERIOD_BETWEEN_SYNCS);
+  }
+}
+
+export async function syncWithKoBo(models, dataBroker) {
+  await pullLatest(models, dataBroker);
+}
+
+async function pullLatest(models, dataBroker) {
+  const syncCursor = await models.syncCursor.findOne({ service_name: SERVICE_NAME });
+
+  // Pull data from KoBo
+  const koboData = await dataBroker.pull(
+    {
+      code: syncCursor.config.koboSurveys,
+      type: dataBroker.getDataSourceTypes().SYNC_GROUP,
+    },
+    {
+      startSubmissionTime: reformatDateStringWithoutTz(syncCursor.sync_time),
```

Comment:
```suggestion
      startSubmissionTime: syncService.sync_cursor,
```
Strangely I think using a worse column type (`TEXT`) for the sync cursor will make our jobs easier here - we just save what comes back, and know it won't have been [mis]interpreted and changed at any steps along the way